.github/workflows/ci.yml

```diff
@@ -6,6 +6,17 @@ on:
   pull_request:
   workflow_dispatch:
 jobs:
+  static-site-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - run: cd static-site/
+      - run: npm ci
+      - run: npm run lint
```

Comment:
🤦 This wasn't working as expected because `cd` only applies to the current step. Need to use `working-directory`.

Reply:
Good catch! Relevant [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun):

> Each run keyword represents a new process and shell in the runner environment. 

Looks like you can also set a working directory for all steps in the job (although in this case one line more than your solution!)

```yaml
  static-site-lint:
    runs-on: ubuntu-latest
    defaults:
      run:
        working-directory: static-site
```

Reply:
Oh, I like the `defaults` approach better. Even though it's more lines, it makes more sense because the scope of the directory is on the job level, not step level.

Reply:
Updated in [force-push](https://github.com/nextstrain/nextstrain.org/compare/6083dc569735afc877bb85cffbd85a1b6578c406..225accdf503a35290769b5e0caef57704b39e236).